tutorials/iot/ubuntu-kiosk/wayland-kiosk.md

```diff
@@ -533,39 +551,40 @@ We will revisit this later to ensure the snap can be compiled to function on oth
 
 duration: 4:00
 
-We need to set up the symlink of the Wayland socket into the $XDG_RUNTIME_DIR directory, and set the libgl environment variable before executing the binary inside the snap. The easiest option is using a simple launching script:
-
+We need to set up the symlink of the Wayland socket into the $XDG_RUNTIME_DIR directory, and set the libgl environment variable before executing the binary inside the snap. The easiest option is using `mir-kiosk-snap-launch` a utility maintained by the Mir team to make this easy. Add the following to the .yaml in the `parts` section:
 
-```bash
-#!/bin/bash
-mkdir -p $XDG_RUNTIME_DIR
-ln -sf $XDG_RUNTIME_DIR/../wayland-0 $XDG_RUNTIME_DIR/
-$SNAP/usr/bin/glmark2-wayland --fullscreen
+```yaml
+  mir-kiosk-snap-launch:
+    plugin: dump
+    source: https://github.com/MirServer/mir-kiosk-snap-launch.git
+    override-build:  $SNAPCRAFT_PART_BUILD/build-with-plugs.sh opengl wayland
 ```
 
 
-…and point `command:` in the snapcraft.yaml file to it (don't forget to install the script in the snap!).
+…and change `command:` in the snapcraft.yaml file to it (don't forget to install the script in the snap!).
```

Comment:
Since it's `dump`ed, does "don't forget…" apply still?

Reply:
Fixed